test/controllers/csvs_controller_test.rb

```diff
@@ -0,0 +1,335 @@
+require_relative '../test_helper'
+
+describe CsvsController do
+  let(:deployer) { users(:deployer) }
+  let(:export) { csv_exports(:pending) }
+
+  describe "permissions test" do
+    as_a_admin do
+      describe "a GET to new" do
+        it "renders the page with user link & admin menu" do
+          get :new
+          assert_template :new
+          assert @response.body.include? "Create Deploys CSV Report"
```

Comment:
use `@response.body.must_include` / `wont_include` to get nice error message when it fails instead of `assertion failed`